### ISSUE TITLE
Moneybag icon

### ISSUE THREAD

**Issue by seddik** [Templarian/MaterialDesign#5051]
I have:

- [x] [Searched all issues](https://github.com/Templarian/MaterialDesign/issues) to make sure there isn't a request for this icon.
- [x] [Searched the current library](https://materialdesignicons.com/) to make sure the icon doesn't exist.
- [x] Only requested a single icon (or a few near-identical ones) in this issue. 

## Usage

- Payments
- Financial stuff

## Examples

![image](https://user-images.githubusercontent.com/6674347/79676990-fcf9a580-81e3-11ea-8792-6b61309376b4.png)

**Comment by Templarian** [comment]
Duplicate of #338

We don't add currency symbols to icons as it makes internationalization difficult.

**Comment by seddik** [comment]
@Templarian in my opinion, that's not a reason to ignore theses types of icon, you already have home-currency_usd and cash-usd. an empty sack is meaningless. 

I see a currency moneybag is a must.

**Comment by Templarian** [comment]
That one was added before we started enforcing this policy after 3.x. While it's a useful icon it's not one we can nicely support with all the various currency symbols.

Try and look for other icons in the set that might work better. There are some more generic cash icons. Like `account-cash` to show a persons financials.
